jupyterhub/utils.py

```diff
@@ -661,58 +661,34 @@ def utcnow():
 
 def _parse_accept_header(accept):
     """
-    Parse the Accept header *accept*
-
-    Return a list with 3-tuples of
-    [(str(media_type), dict(params), float(q_value)),] ordered by q values.
-    If the accept header includes vendor-specific types like::
-        application/vnd.yourcompany.yourproduct-v1.1+json
```

Comment:
Was this originally for some legacy browsers that sent unusual headers, or is it just old code copied from somewhere else that was never needed?